README.md

```diff
@@ -1,10 +1,20 @@
-# Eligibility rules service exemplar - Research project
+# Eligibility rules service - Phase 2 (research/discovery)
 
-## What is this?
+## What is this project?
 
 18F is conducting research to identify opportunities for developing a web-based eligibility rules service that could be used to make eligibility determinations for a human services program. This _Phase 2_ work follows a prior _Phase 1_ investigation sprint that identified creating such a service as worthy of further exploration. This repository documents Phase 2.
 
-The hypothesis: Both federal and state agencies could achieve greater program efficiencies if the administering federal agency offered a configurable, API-based service that allowed state partner to make eligibility determinations, rather than building and maintaining their own separate rules engines.
+**The problem:**
```

Comment:
I figure we'll want to edit this a bit, but wanted to see your thoughts about adding a bit more color about the problem we are trying to solve, as well as about our hypothesis. If you think this belongs in a wiki vs in the readme, I'm cool with that, but I figured we had a lot of thoughtful language already in our pitch that we could pull from, and wanted to try out putting it here.